app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+class Animal:
+
+    alive = []
+
+    def __init__(
+            self,
+            name: str,
+            health: int = 100,
+            hidden: bool = False
+    ) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        info_anim = f"Name: {self.name}," \
+                    f" Health: {self.health}," \
+                    f" Hidden: {self.hidden}"
+        return "{" + f"{info_anim}" + "}"
```

Comment:
Use double "{{" "}}" inside f-string, it will represent as single "{" "}". And remove the unnecessary variable, Just return `f"{{...}}"`